test/omise/dispute_test.exs

```diff
@@ -2,11 +2,8 @@ defmodule Omise.DisputeTest do
   use ExUnit.Case
```

Comment:
According to https://github.com/jjh42/mock (the package I use for mocking):

> Also, note that Mock has a global effect so if you are using Mocks in multiple tests set async: false so that only one test runs at a time.